tests/PHPStan/Analyser/data/bug-5219.php

```diff
@@ -7,9 +7,9 @@
 class HelloWorld
 {
 
-	protected function foo(string $message): void
+	protected function foo(string $message, string $x): void
```

Comment:
adjusted the existing test, so it still leads to the same result as before this PR.